src/core.js

```diff
@@ -392,6 +393,99 @@ define([
         ];
     };
 
+    fn._init_bulk_update_questions = function () {
```

Comment:
I know we discussed this earlier, but now that I'm seeing it in code, it does seem it may be nicer to implement this as a new plugin rather than adding another ~100 lines to core.js (which is already way too big).

Reply:
yeah...this is why i had asked right before starting this work...but i guess it's not much more of an extra lift. It will certainly delay things going into QA though vs having done it as a plugin from the start. FYI Dev

Reply:
@millerdev What is the fundamental concern associated with the current extra 100 lines of code to core.js? For example, does it negatively affect code maintainability? or are there other concerns like performance issues.

Reply:
> does it negatively affect code maintainability

Yes, this. The file is already huge, and adding more stuff in here makes it harder to navigate and find things within the file.

Reply:
Got it. thanks.

Reply:
Also, I think it would be fine to do this as a follow-up PR or while QA is in progress.